TeamCharles/initial-site
# 89-addToCartButton

## Status

**READY**
## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.
## Description

When a user is signed in, a user can add a product to his/her cart.
## Resolves Issue Number

https://github.com/TeamCharles/initial-site/issues/89
## Deploy Notes

``` sh
git pull --prune
git checkout 89-addToCartButton
dotnet run
```
## Steps to Test or Reproduce
1. Click the `Cart` button in the Nav bar. This is the only way to log into an account currently.
2. Click on the `Bangazon` title in the Nav bar and find a product to add to your cart.
3. Click the `Add to Cart` button.
4. If successful, it should reroute you back to the `Detail/{ProductId}` route (the page you were already viewing)
5. Click the `Cart` button to see the new item added to `Carson`'s order.
## Impacted Files in Application
- Controllers/CartController.cs
- Views/Products/Detail.cshtml
